common/lib/xmodule/xmodule/js/src/video/09_events_plugin.js

```diff
@@ -61,7 +61,9 @@ define('video/09_events_plugin.js', [], function() {
         },
 
         onPlay: function () {
-            this.log('play_video', {currentTime: this.getCurrentTime()});
+            if (this.state.videoPlayer.emitPlayVideoEvent) {
```

Comment:
@muhammad-ammar Why do we need two new variables? Can we just not make this line the following?

```
if (this.videoPlayer.previousPlayerState !== this.videoPlayer.PlayerState.PLAYING)
```